crates/bevy_gltf/src/loader.rs

```diff
@@ -497,6 +498,16 @@ async fn load_gltf<'a, 'b>(
                 .map(|node| node_index_to_entity_map[&node.index()])
                 .collect();
 
+            if joint_entities.len() > MAX_JOINTS && warned_about_max_joints.insert(skin_index) {
```

Comment:
Does this properly short circuit, and not insert into the hashmap if the len <= MAX_JOINTS?

Reply:
yes that's how `&&` works in rust, see <https://doc.rust-lang.org/reference/expressions/operator-expr.html#lazy-boolean-operators>